include/string.cpp

```diff
@@ -0,0 +1,223 @@
+// Copyright 2020 Merzlov Nikolay merzlovnik@mail.ru
+#include "string.hpp"
+
+String::~String() {
+    delete[] Data;
+}
+
+String::String() : size(0) {
+    Data = new char[size];
```

Comment:
Initialize `Data` variable using `[varname]([value])` syntax (as done for `size`)
Also, `size` is always `0`

Reply:
Even better, add a non-`public` constructor `String(size_t)` which will initialize `size` and `Data`